packages/web-service/src/pages/error/server-error.njk

```diff
@@ -1,6 +1,10 @@
 {% extends "information-page.njk" %}
 {% set title = 'Sorry, there is a problem with the service' %}
-
 {% block pageContent %}
```

Comment:
Because this file (`server-error.njk`) is now going to serve all 4xx and 5xx errors, I think we need to rename it, because it isn's just server-errors now. It could be client errors

Reply:
resolved